telemetry_api/lib/telemetry_api/periodically.ex

```diff
@@ -24,7 +25,9 @@ defmodule TelemetryApi.Periodic.OperatorFetcher do
   end
 
   def send_work() do
+    one_second = 1000
     :timer.send_interval(@wait_time_ms, :fetch_operators)
+    :timer.send_interval(one_second * 12, :gas_price) # every 12 seconds, once per block
```

Comment:
I would recommend once per 10 seconds, for redundancy, as not every block is exactly 12 seconds.

```suggestion
    :timer.send_interval(one_second * 10, :gas_price) # every 10 seconds, once per block with a bit of some margin
```